src/utils/contextUtils.ts

```diff
@@ -28,9 +28,12 @@ export async function loadPackageInfo(context: ExtensionContext): Promise<void>
 
     TEMP_FOLDER_PER_USER = path.join(os.tmpdir(), `${EXTENSION_NAME}-${os.userInfo().username}`);
 
-    // find Maven Local Repositry
+    // find Maven Local Repository

```

Comment:
You should add a listener for this setting. Whenever the value is changed, you should also update value of `MAVEN_LOCAL_REPOSITORY`.
I suggest to extract this part into a function, e.g. `loadMavenSettings()`, and re-use it.

Reply:
OK